src/index.html

```diff
@@ -11,6 +11,28 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+          method="get"
+          data-qa="big">
+      <label for="searchBig">
+        <input type="search"
+               placeholder="Try &quot;Los Angeles&quot;"
+               id="searchBig"
+               data-qa="keypress"
+               class="big-input">
+      </label>
+    </form>
```

Comment:
Go through [checklist](https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md) one more time and fix your code style, look closely at 3rd item